src/Voice.tsx

```diff
@@ -375,6 +375,9 @@ export class Voice extends EventEmitter {
       [Constants.VoiceEventCallInviteRejected]: this._handleCallInviteRejected,
       [Constants.VoiceEventCallInviteCancelled]:
         this._handleCancelledCallInvite,
+      [Constants.VoiceEventCallInviteNotificationTapped]: () => {
+        /* TODO(mhuynh): handle this event */
```

Comment:
Do we need a ticket for this @mhuynh5757 @afalls-twilio?

Reply:
Yeah probably a good idea. I've filed VBLOCKS-2017.